src/velo-fed/BaseFedCCTP.sol

```diff
@@ -166,105 +222,129 @@ contract BaseFedCCTP {
+     * @param newMaxSlippageBps The new maximum allowed loss for DOLA -> USDC swaps. 1 = 0.01%
+     */
+    function setMaxSlippageDolaToUsdc(
+        uint256 newMaxSlippageBps
+    ) external onlyGov {
         if (newMaxSlippageBps > 10000) revert MaxSlippageTooHigh();
         maxSlippageBpsDolaToUsdc = newMaxSlippageBps;
     }
 
     /**
-    @notice Governance only function for setting acceptable slippage when swapping USDC -> DOLA
-    @param newMaxSlippageBps The new maximum allowed loss for USDC -> DOLA swaps. 1 = 0.01%
-    */
-    function setMaxSlippageUsdcToDola(uint newMaxSlippageBps) external {
-        if (msg.sender != gov) revert OnlyGov();
+     * @notice Governance only function for setting acceptable slippage when swapping USDC -> DOLA
+     * @param newMaxSlippageBps The new maximum allowed loss for USDC -> DOLA swaps. 1 = 0.01%
+     */
+    function setMaxSlippageUsdcToDola(
+        uint256 newMaxSlippageBps
+    ) external onlyGov {
         if (newMaxSlippageBps > 10000) revert MaxSlippageTooHigh();
         maxSlippageBpsUsdcToDola = newMaxSlippageBps;
     }
 
     /**
-    @notice Method for `gov` to change `pendingGov` address
-    @dev `pendingGov` will have to call `claimGov` to complete `gov` transfer
-    @param newPendingGov_ Address to be set as `pendingGov`
-    */
-    function setPendingGov(address newPendingGov_) external {
-        if (msg.sender != gov) revert OnlyGov();
-        pendingGov = newPendingGov_;
+     * @notice Method for `gov` to change `pendingGov` address
+     * @dev `pendingGov` will have to call `claimGov` to complete `gov` transfer
+     * @param newPendingGov Address to be set as `pendingGov`
+     */
+    function setPendingGov(address newPendingGov) external onlyGov {
+        pendingGov = newPendingGov;
     }
 
     /**
-    @notice Method for `pendingGov` to claim `gov` role.
-    */
+     * @notice Method for `pendingGov` to claim `gov` role.
+     */
     function claimGov() external {
         if (msg.sender != pendingGov) revert OnlyPendingGov();
         gov = pendingGov;
         pendingGov = address(0);
     }
 
     /**
-    @notice Method for gov to change the chair
-    @param newChair_ Address to be set as chair
-    */
-    function changeChair(address newChair_) external {
-        if (msg.sender != gov) revert OnlyGov();
-        chair = newChair_;
-    }
-
-    /**
-    @notice Method for gov to change the L2 aeroFarmer address
-    @dev aeroFarmer is the L2 address that receives all bridged DOLA from expansion
-    @param newAeroFarmer_ L2 address to be set as aeroFarmer
-    */
-     function changeAeroFarmer(address newAeroFarmer_) external {
-        if (msg.sender != gov) revert OnlyGov();
-        aeroFarmer = newAeroFarmer_;
+     * @notice Method for gov to change the chair
+     * @param newChair Address to be set as chair
+     */
+    function changeChair(address newChair) external onlyGov {
+        chair = newChair;
```

Comment:
Probably a good idea to make an inheritable `Chairable` contract with:

- `onlyGov` modifier
- `onlyChair` modifier
- set pending/accept logic for gov
- `changeChair(address newChair)` function only callable by gov
- `resign` function that sets fed chair address to 0, only callable by chair